draft-ietf-grow-bmp-local-rib.txt

```diff
@@ -557,11 +557,14 @@ Internet-Draft                BMP Local-RIB                    July 2019
 
 
 
-Evens, et al.           Expires January 12, 2020               [Page 10]
+Evens, et al.           Expires January 19, 2020               [Page 10]
 
 Internet-Draft                BMP Local-RIB                    July 2019
 
 
+   Multiple TLVs of the same type can be repeated as part of the same
+   message, for example to convey a filtered view of a VRF.
```

Comment:
The use case you mentioned was to reference both VRF name and Table name, hence two TLV's of the same type.  In this sense, we should maybe call this out.  For example:

"A BMP receiver should append to a set multiple TLVs of the same type.  This is to support alternate or additional names for the same peer.  For example, a VRF name Blue and table name "customer-abc" could be conveyed."

Reply:
We should also call out order...    IMO, the order should be maintained in the Set (e.g. Ordered Set).  I use Set instead of List because there should be no duplicates.  If there is a duplicate, it should be replaced by the last one.   Standard append to a Set.